chart/values.yaml

```
@@ -319,7 +319,24 @@ extraEnvFrom: ~
 # Airflow database & redis config
 data:
   # If secret names are provided, use those secrets
+  # These secrets must be created manually, eg:
+  #
+  # kind: Secret
+  # apiVersion: v1
+  # metadata:
+  #   name: custom-airflow-metadata-secret
+  # type: Opaque
+  # data:
+  #   connection: base64_encoded_connection_string
+
   metadataSecretName: ~
+  # When providing secret names and using the same database for metadata and
+  # result backend, for Airflow < 2.4.0 it is necessary to create a separate
+  # secret for result backend but with db+ prefix in connection. For example:
+  # data:
+  #   connection: base64_encoded_connection_string_with_db+_prefix
+  # For airflow >= 2.4.0 it is possible to not specify the secret,
+  # connection from metadataSecretName with db+ prefix will be used by default
```

```suggestion
  # For Airflow >= 2.4.0 it is possible to not specify the secret again,
  # as Airflow will use sql_alchemy_conn with a db+ scheme prefix by default.
```
I think this reads better.